src/test/java/com/apruve/models/ApruveModelTestHelper.java

```diff
@@ -0,0 +1,12 @@
+package com.apruve.models;
+
+import com.apruve.JsonUtil;
+
+public class ApruveModelTestHelper {
+
+	protected static String doMarshalTest(Object obj) {
```

Comment:
Is the point here just to make sure it doesn't throw any errors? Or are we missing something that tests that it marshals back to a reasonable representation?

Reply:
Yes, it's looking for configuration problems which would throw exceptions from the serialization framework.  Though I realize in trying to DRY out the code I inadvertently switched to a serialization method that intentionally fails without throwing an exception.  Whoops.